gapir/cc/resource_cache.h

```diff
@@ -22,6 +22,8 @@
 #include <memory>
 #include <vector>
 
+#include "replay_connection.h"
+
 namespace gapir {
 
 // ResourceCache is an abstract base class for caching resource providers.
```

Comment:
... in fact all of these cache classes could be massively simplified now.

Reply:
TODO added